lib/handlers/pageHandler.js

```diff
@@ -1,5 +1,5 @@
 const { createJsDialogEventName } = require('../util');
-const { handleUrlRedirection } = require('../helper');
+const { handleUrlRedirection, isElement } = require('../helper');
 const { eventHandler } = require('../eventBus');
 const { logEvent } = require('../logger');
 const { isSelector, findElements } = require('../elementSearch');
```

Comment:
Can we use the `isSelector` from [helper.js](https://github.com/getgauge/taiko/blob/374ad9d7ac1c987e2d1c7c1cb98426ce44e06b98/lib/helper.js#L207). 
Please remove the re-export of `isSelector` from [elementSearch.js](https://github.com/getgauge/taiko/blob/374ad9d7ac1c987e2d1c7c1cb98426ce44e06b98/lib/elementSearch.js#L325)